client/homebrew/editor/snippetbar/snippetbar.jsx

```diff
@@ -138,6 +149,35 @@ const Snippetbar = createClass({
 		});
 	},
 
+	replaceContent : function(item){
+		return this.props.updateBrew(item);
+	},
+
+	renderHistoryItems : function() {
+		const historyItems = getHistoryItems(this.props.brew);
+
+		return <div className='dropdown'>
+			{_.map(historyItems, (item, index)=>{
+				if(!item.savedAt) return;
+
+				const saveTime = new Date(item.savedAt);
+				const diffMs = new Date() - saveTime;
+				const diffMins = Math.floor(diffMs / (60 * 1000));
+
+				let diffString = `about ${diffMins} minutes ago`;
```

Comment:
Can we have seconds too? It was throwing me off seeing "0 minutes" the whole time.

Reply:
Should be added now.